lib/pages/login/widget/login_form.dart

```diff
@@ -37,6 +37,7 @@ class _LoginFormState extends ConsumerState<LoginForm> {
       mainAxisAlignment: MainAxisAlignment.center,
       children: [
         TextFormField(
+          key: Key('tfLoginEmail'),
```

Comment:
Can we make a constant here so we can reuse it in the test? 🙏 
```dart
class LoginForm extends ConsumerStatefulWidget {
    static const loginEmailKey = Key('tfLoginEmail')

// test
final Finder emailField = find.byKey(LoginForm.loginEmailKey);
```

Reply:
@markgravity Sure, I fixed it in the next PR https://github.com/luongvo/flutter-survey/pull/93/commits/529175e901a1a44d918229b4d5ae2b73a25ab9f1 ✅